Vipul Kumar/if else/03_discount.cpp

```diff
@@ -0,0 +1,22 @@
+#include<iostream>
+using namespace std;
+
+int main()
+{
+int quantity;
+int cost;
+cin>>quantity;
+if (quantity>1000)
```

Comment:
it quantity*100 > 1000, as cost is given as 100 and we need to check total cost is more than 1000 or not, we don't have to check total quantity is more than 1000 or not.